content/en/docs/plugins/_index.md

```diff
@@ -129,17 +131,16 @@ Falco configures plugins via the new "plugins" property in falco.yaml. Here's an
 
 ```yaml
 plugins:
-  - name: aws_cloudtrail
-    library_path: aws_cloudtrail/plugin.so
-    init_config: "..."
-    open_params: "..."
-  - name: http_json
-    library_path: http_json/plugin.so
-    init_config_file: http_json/config.txt
-    open_params_file: http_json/params.txt
+  - name: cloudtrail
+    library_path: libcloudtrail.so
+    init_config: ""
+    open_params: ""
+  - name: json
+    library_path: libjson.so
+    init_config: ""
 
 # Optional
-load_plugins: [aws_cloudtrail]
+load_plugins: [cloudtrail]
```

Comment:
```suggestion
load_plugins: [cloudtrail, json]
```
Does `cloudtrail` work without `json` ?

Reply:
Well the plugin operates but the rules in aws_cloudtrail_rules.yaml won't. I think we should include both to be a realistic example.